etc/kayobe/environments/ci-multinode/network-allocation.yml

```diff
@@ -0,0 +1,2 @@
+external_ips:
+  alex-mn-seed: 192.168.38.3
```

Comment:
This isn't going to work if you change the cluster name...

Reply:
yeah, painfully obvious once I actually thought about it. Is there some magic way of getting the name of the seed in Kayobe or will it need to be terraform templated?

Reply:
Possibly you could use `{{ groups['seed'][0] }}` as the key, but I don't think the ip-allocation playbook would handle this.

I think currently terraform does not know about the IP subnets in this config, and it should probably stay that way.

Alternatively we could rely on the fact that `kayobe seed host configure` runs first and will grab this IP.

Reply:
> Possibly you could use `{{ groups['seed'][0] }}` as the key, but I don't think the ip-allocation playbook would handle this.
I gave that a go but it doesn't seem to template properly. Surprisingly it didn't error out but it also didn't apply the change. 

I think we may just have to assume it's on `192.168.38.3` and change either the kayobe or openstack configuration if the environment is different